middleware/compress.go

```diff
@@ -29,6 +29,7 @@ var defaultContentTypes = map[string]struct{}{
 	"application/json":         struct{}{},
 	"application/atom+xml":     struct{}{},
 	"application/rss+xml":      struct{}{},
+        "image/svg+xml":            struct{}{},
```

Comment:
thx. can you fix the indentation here? its breaking the CI due to formatting

Reply:
It's very surprising that there is any kind of issue as it ran through `go fmt` before pushing. Will check it out.